inc/class-cli-command.php

```diff
@@ -78,6 +78,141 @@ public function delete( $args, $assoc_args ) {
 			}
 		}
 	}
+
+	/**
+	 * Set id as primary key in the Native PHP Sessions plugin table.
+	 *
+	 * @subcommand add-index
+	 */
+	public function add_index( $args, $assoc_arc ) {
+		global $wpdb;
+		$unprefixed_table = 'pantheon_sessions';
+		$table            = $wpdb->base_prefix . $unprefixed_table;
+		$temp_clone_table = $wpdb->base_prefix . 'sessions_temp_clone';
+
+		// If the command has been run multiple times and there is already a
+		// temp_clone table, drop it.
+		$query = $wpdb->prepare( 'SHOW TABLES LIKE %s', $wpdb->esc_like( $temp_clone_table ) );
+
+		if ( $wpdb->get_var( $query ) == $temp_clone_table ) {
+			$query = "DROP TABLE {$temp_clone_table};";
+			$wpdb->query( $query );
+		}
+
+		if ( ! PANTHEON_SESSIONS_ENABLED ) {
```

Comment:
```suggestion
		if ( ! PANTHEON_SESSIONS_ENABLED ) {
```
```suggestion
		if ( ! defined("PANTHEON_SESSIONS_ENABLED") || ! PANTHEON_SESSIONS_ENABLED ) {
```
Someone confirm this is behavior we want if constant DNE.

Reply:
@greg-1-anderson @jazzsequence  Can either of you weigh in here?

Reply:
I searched and see now this is defined when the plugin is loaded so nevermind.